.github/workflows/ci.yaml

```diff
@@ -104,16 +109,16 @@ jobs:
         if: matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-latest'
         run: ./tools/install_deploy_package_to_data_dir.sh
 
-      - name: Build external dependencies 
-        run: cabal build --enable-tests --enable-benchmarks --only-dependencies 
-      
+      - name: Build external dependencies
+        run: cabal build --enable-tests --enable-benchmarks --only-dependencies
+
       - name: Build wasp code
         run: cabal build all
 
       - name: Set up Node
         uses: actions/setup-node@v3
         with:
-          node-version: '18'
+          node-version: '18.12'
```

Comment:
You pinned this down to make sure it works now, due to it pulling too new version of `npm`, 9 instead of 8?

While this is ok as a temporary fix, I would consider putting it back to `18` if we decide to drop the restrictions on `npm`, because we want to have the same experience as users have. It is good that we got the error in this case, when users also got it, I think we want to keep this behaviour.

Reply:
I'll update the version when we fix the issue of using Node.js 18.14 👍

Reply:
Ok! For the future, it is always good to leave the TODO if you have future plans or something is temporary -> you could forget for some reason, and then we would have this here but wouldn't know we should change it or why it was done this way.